lib/config.js

```diff
@@ -32,11 +32,23 @@ function expandTilde(apath) {
     return path.join(homedir, apath.slice(2));
```

Comment:
I like solution 1 the best. Its easiest to understand and setup for a user, least lines of code needed, and easiest to maintain as part of the codebase.